tasks.py

```diff
@@ -302,3 +303,32 @@ def reset_all():
     os.system("curl -XDELETE '{}/share*'".format(settings.ELASTIC_URI))
     os.system("invoke alias share share_v2")
     os.system("invoke provider_map")
+
+
+@task
+def institutions(grid_file='institutions/grid_2015_10_09.json', ipeds_file='institutions/hd2013.csv'):
+    grid(grid_file)
+    ipeds(ipeds_file)
+
+
+@task
+def remove_institutions(skip=False):
+    if not skip:
+        resp = six.moves.input('You are about to delete the institutions index. Are you sure? (y, n)\n')
+        if resp not in ('y', 'Y', 'Yes', 'yes'):
+            print('Remove institutions stopped.')
+            return
+    from institutions.institutions import remove
+    remove()
+
+
+def grid(grid_file):
```

Comment:
Doesn't this make it so you can't `inv grid` with out a file? also, I'd call it `filepath`, so you don't have to remember whether it's `-i` or `-g`